src/components/Oppgave/KollapsbarOppgavetekstContainer.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import styles from "./Oppgave.module.css";
+import { StatusType } from "./Oppgave";
+
+function erKollapsetType(status?: StatusType): boolean {
```

Comment:
Foretrekker kanskje litt mer deklarativ navngivning, for eksempel  `skalVisesKollapset()`